packages/forma-36-codemod/utils/changeImport.js

```diff
@@ -4,15 +4,20 @@ const { removeComponentImport } = require('./removeComponentImport');
 module.exports.changeImport = function changeImport(
   j,
   source,
-  { from, to, componentName },
+  { from, to, componentName, outputComponentName },
 ) {
   let result = removeComponentImport(j, source, {
     componentName,
     importName: from,
   });
 
   const importResult = addImport(j, result, [
-    j.template.statement([`import { ${componentName} } from "${to}"`]),
+    j.template.statement([
+      `import { ${
+        outputComponentName ? outputComponentName : componentName
+      } } from "${to}"`,
```

Comment:
Wouldn't this remove other imports, in case more than one component is being imported from that package?

Reply:
☝️  That's what wanted to ask in my other comment hahaha
Massao just did it better 👏

Reply:
Got it now. Sorry, didn't get it what you mean at the beginning. Yup, good point, I will check it out.